### PR TITLE
Added AutoRCM

### PR DESCRIPTION
**Pull request by Reisyukaku**
Was easier to delete old repo and retry.. I screwed up that bad.

### CODE REVIEW

File: ipl/main.c [hunk 1]
```diff
@@ -1159,6 +1159,37 @@ out:;
...
+		sect = (0x200 + (0x4000 * i)) / NX_EMMC_BLOCKSIZE;
+		sdmmc_storage_read(&storage, sect, 1, tempbuf);
+		tempbuf[0x10] ^= 0x77; //xor by arbitrary number to corrupt
+		sdmmc_storage_write(&storage, sect, 1, tempbuf);
+	}
+	
+	free(tempbuf);
+	sdmmc_storage_end(&storage);
+	
+	gfx_printf(&gfx_con, "%kAutoRCM mode toggled!", 0xFF00EE2C);
```
Comment (CTCaer) [inline ipl/main.c]: `gfx_printf(&gfx_con, "%kAutoRCM mode toggled!", 0xFF00EE2C);`

should be 

`gfx_printf(&gfx_con, "%kAutoRCM mode toggled!%k\n", 0xFF00EE2C, 0xFFFFFFFF);`

File: ipl/main.c [hunk 2]
```diff
@@ -1211,15 +1242,36 @@ menu_t menu_cinfo = {
...
+ment_t ment_autorcm[] = {
+	MDEF_BACK(),
+	MDEF_BACK(),
+	MDEF_BACK(),
+	MDEF_BACK(),
+	MDEF_HANDLER("Toggle AutoRCM", toggle_autorcm),
+	MDEF_BACK(),
+	MDEF_BACK(),
+	MDEF_BACK(),
+	MDEF_BACK(),
```
Comment (CTCaer) [inline ipl/main.c]: Remove one `MDEF_BACK(),` after the `MDEF_HANDLER` for symmetry?

File: ipl/main.c [hunk 3]
```diff
@@ -1211,15 +1242,36 @@ menu_t menu_cinfo = {
...
+};
+
 ment_t ment_tools[] = {
 	MDEF_BACK(),
 	MDEF_HANDLER("Dump RAW eMMC", dump_emmc_rawnand),
 	MDEF_HANDLER("Dump eMMC SYS", dump_emmc_system),
 	MDEF_HANDLER("Dump eMMC USER", dump_emmc_user),
 	MDEF_HANDLER("Dump eMMC BOOT", dump_emmc_boot),
 	MDEF_HANDLER("Dump package1", dump_package1),
+	MDEF_MENU("AutoRCM", &menu_autorcm), //Menu
```
Comment (CTCaer) [inline ipl/main.c]: Remove the comment ( //Menu), it was to explain what to do.

**Comment by CTCaer** [comment]
Previous discussion: https://github.com/nwert/hekate/pull/19

**Comment by Reisyukaku** [comment]
Fair enough, done.

**Pull request merged**